test/black-box/RpcGetItem.test.ts

```diff
@@ -111,7 +111,9 @@ describe('/listing-items', () => {
         await command.run();
     });
 
-    test('POST      /listing-items        Should create a new listing item', async () => {
+
+    test('Should get the listing item by hash and id', async () => {
+        // create listing item
         const res = await api('POST', '/api/listing-items', {
```

Comment:
again this is not testing anything, just inserting test data, move that to the test below.